src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Intake {
+    private CANSparkMax verticalRollerMotor;
+    private CANSparkMax horizontalRollerMotor;
+    private CANSparkMax deploymentMotor;
+    private double suckSpeed;
+    private double spitSpeed;
+    private double inactiveSpeed;
+  
+    public Intake() {
+      // Initializing motor controllers
+      verticalRollerMotor = new CANSparkMax(0, MotorType.kBrushless);
+      horizontalRollerMotor = new CANSparkMax(0, MotorType.kBrushless);
+      deploymentMotor = new CANSparkMax(0, MotorType.kBrushless);
+  
+      // Motor speeds
+      suckSpeed = 1.0;
+      spitSpeed = -0.3;
+      inactiveSpeed = 0.0;
+    }
+
+    //TODO: for the deploy and intake methods, use position control to accurately set motors based on encoder position
+
+    //deploy intake
+    public void deploy(boolean spit, double deploymentMotorSpeed) {
+      if (spit) {
+        verticalRollerMotor.set(spitSpeed);
+        horizontalRollerMotor.set(spitSpeed);
+      } else {
+        verticalRollerMotor.set(suckSpeed);
+        horizontalRollerMotor.set(suckSpeed);
+      }
+  
+      deploymentMotor.set(deploymentMotorSpeed);
+    }
+  
+    // stow intake
+    public void stow() {
+      verticalRollerMotor.set(inactiveSpeed);
+      horizontalRollerMotor.set(inactiveSpeed);
+      deploymentMotor.set(inactiveSpeed);
```

Comment:
Right now this wouldn't stow because you're giving setting the deployment motor to 0 (so they don't move). Once we add PID, we won't have this issue, but (for now) I'd still recommend that you make it so that the intake and deployment are handled with separate buttons. This way the operator could deploy the intake, wait till it hits the bottom, then start intaking.

Reply:
![image](https://user-images.githubusercontent.com/68178572/197251429-11d954c1-991e-45ae-a550-43fd425c7e4c.png)

Reply:
Also, in terms of controls, shall I just use two face buttons instead of the right trigger as I have it implemented?

Reply:
So, while you could let it stall, I wouldn't do this (especially if the input is constant). Again, PID will fix this problem but (for now) it's probably best to put the deploy motor in brake mode and have the operator control it separately from the intake so that the intake is not constantly trying to close itself. Also, right now the deploymentMotor is still set to the inactive speed of 0? This would mean that right now it isn't trying to close itself.

Reply:
Alright cool, thanks for the feedback.